common/commands/config.go

```diff
@@ -174,6 +176,9 @@ func (cc *ConfigCommand) config() error {
 		// Some package managers support basic authentication only.
 		// To support them we try to extract the username from the access token
 		if cc.details.AccessToken != "" && cc.details.User == "" {
+			if httpclient.IsApiKey(cc.details.AccessToken) {
+				return errors.New("the provided Access Token is an API key and should be used as a password in username/password authentication")
```

Comment:
Since this error message is duplicated twice in this file, let's move this `errors.New` function into a new function named `newApiKeyUsedAsTokenError()` function and reuse it.